Packs/McAfeeNSM/Integrations/McAfeeNSMv2/McAfeeNSMv2.py

```diff
@@ -1831,19 +2124,321 @@ def export_pcap_file_command(client: Client, args: Dict) -> List:
+        raise DemistoException('Please provide a domain_id.')
+    pre_firewall_policy = args.get('pre_firewall_policy_name')
+    post_firewall_policy = args.get('post_firewall_policy_name')
+
+    response = client.assign_device_policy_request(domain_id=domain_id, device_id=device_id,
+                                                   pre_firewall_policy=pre_firewall_policy,
+                                                   post_firewall_policy=post_firewall_policy
+                                                   )
+    readable_output = 'Policy assigned successfully.' if response.get('status') == 1 else 'Policy assignment failed.'
+    return CommandResults(
+        readable_output=readable_output,
+        raw_response=response)
+
+
+def list_device_policy_command(client: Client, args: Dict) -> CommandResults:
+    """
+    Retrieves a list of policies related to the domain or a specific device.
+    Args:
+        client(Client): client - A McAfeeNSM client.
+        args(Dict): - The function arguments.
+
+    Returns:
+         A CommandResult object with a list of device policies.
+    """
+    device_id = arg_to_number(args.get('device_id'))
+    domain_id = arg_to_number(args.get('domain_id'))
+    if not domain_id and domain_id != 0:
+        raise DemistoException('Please provide a domain_id.')
+    limit = arg_to_number(args.get('limit', DEFAULT_LIMIT))
+    all_results = argToBoolean(args.get('all_results', False))
+
+    response = client.list_device_policy_request(domain_id=domain_id, device_id=device_id)
+    all_policies = response.get('policyAssignmentsList', [])
+
+    capitalize_policies = capitalize_key_first_letter(all_policies) if all_results else \
+        capitalize_key_first_letter(all_policies)[:limit]
+
+    readable_output = tableToMarkdown(
+        name='Device policy List', t=capitalize_policies, removeNull=True
+    )
+
+    return CommandResults(
+        readable_output=readable_output,
+        outputs_prefix='NSM.DevicePolicy',
+        outputs=capitalize_policies,
+        raw_response=response
+    )
+
+
+def assign_interface_policy_command(client: Client, args: Dict) -> CommandResults:
+    """
+    Assigns an existing policy to an interface.
+    Args:
+        client(Client): - A McAfeeNSM client.
+        args(Dict): - The function arguments.
+    Returns:
+        A CommandResult object with a success or failure message.
+    """
+    domain_id = arg_to_number(args.get('domain_id'))
+    if not domain_id and domain_id != 0:
+        raise DemistoException('Please provide a domain_id.')
+    interface_id = arg_to_number(args.get('interface_id'))
+    if not interface_id and interface_id != 0:
+        raise DemistoException('Please provide a interface_id.')
+    firewall_policy = args.get('firewall_policy_name')
+    firewall_port_policy = args.get('firewall_port_policy_name')
+    ips_policy = args.get('ips_policy_name')
+    custom_policy_json: str = args.get('custom_policy_json') or ""
```

Comment:
Why not using:
```suggestion
    custom_policy_json: str = args.get('custom_policy_json', '')
```
Is it because of mypy?

Also, why do we need the str declaration here?

Reply:
@RosenbergYehuda

Reply:
mypy